Components/Toasts/toast.css

```diff
@@ -0,0 +1,48 @@
+.toast{
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    /* width: 100%; */
+    max-width: 500px;
+    margin: 0px 1rem 0px 1rem;
+    font-size: 1rem; 
+    padding: 1rem;
+    border-radius: 0.4rem;
+    z-index: 99;
```

Comment:
Above you have given navbar z-index : 100, which is fine because you want it on top of everything else. Then here you gave z-index : 99, for simplicity you can keep z-index till 10 or 15. I don't think you would need 100 z-indexes.

Reply:
Ya it would also work, i will change😊